libs/blocks/accordion/accordion.js

```diff
@@ -10,7 +11,40 @@ function setSEO(questions) {
   document.head.append(script);
 }
 
-function handleClick(el, dd) {
+function displayMedia(displayArea, el, dd, i, id, expanded) {
+  const theID = el.getAttribute('aria-controls').split('-')[1];
+  [...mediaCollection[theID]].map(
+    (mediaCollectionItem, idx, total) => {
+      mediaCollectionItem.classList.remove('expanded');
+
+      for (let index = 0; index < total.length; index++) {
+        mediaCollectionItem.classList.remove('expanded');
```

Comment:
Is this line needed? I think we're already removing the `expanded` class on [L#18](https://github.com/adobecom/milo/pull/942/files#diff-1303b6fa79bd2416fc5d00eae1b6490f4bc5e614f2555824c934df2c05df1ea9R18).